packages/web-pkg/src/components/sidebar/SideBar.vue

```diff
@@ -168,22 +170,13 @@ export default defineComponent({
       },
       immediate: true
     },
-    loading: {
-      handler() {
+    sidebarClosed: {
```

Comment:
erm... we only need to init the visibility observers when the sidebar is being opened, right? from my understanding the watcher should look like this:

```js
sidebarClosed: {
      handler: function (closed) {
        if (closed) {
          return
        }
        this.$nextTick(() => {
          this.initVisibilityObserver()
        })
      },
      immediate: true
    }
```